doc/source/usage.rst

```diff
@@ -59,7 +59,7 @@ A ``.travis.yml`` testing a role named foo1 with the Docker driver.
     - sudo apt-get -qq update
     - sudo apt-get install -o Dpkg::Options::="--force-confold" --force-yes -y docker-engine
   install:
-    - pip install molecule
+    - pip install molecule docker
```

Comment:
Ok, I changed commit. Are you think that anyone will use travis with vagrant or openstack?